otel/otel.sh

```diff
@@ -0,0 +1,79 @@
+# This initialization action installs OpenTelemetry Collector Contrib (otelcol-contrib) in a Dataproc cluster
```

Comment:
We need to add `#!/usr/bin/env bash` on top of this file.

Reply:
Added.